examples/analog_hamiltonian_simulation/01_Introduction_to_Aquila.ipynb

```diff
@@ -7,175 +7,205 @@
     "tags": []
```

Comment:
<p>To support the "We see" part, let's actually create a bar plot for 3-atom states, aggregated across all triangles.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>For now, we can't actually run the program, because it has 100+ atoms. But let me run the program with just one triangle and put the figure as a place holder. </p>